mantidproject/mantid
# Feature/11058 refl new geometry

Fixes issue  [#11058](http://trac.mantidproject.org/mantid/ticket/11058).

To test ticket:
In order to make sure the old and new geometry works, one can use LoadEventNexus and load REF_L run 101101 (old geometry) and 118467 (new geometry). A nice horizontal signal should show up in the View Instrument window.
